src/main/scala/chisel3/package.scala

```diff
@@ -310,6 +310,11 @@ package object chisel3 {    // scalastyle:ignore package.object.name
     val withClock = chisel3.core.withClock
     val withReset = chisel3.core.withReset
 
+    type BaseModule = chisel3.core.BaseModule
+    type MultiIOModule = chisel3.core.ImplicitModule
+    type RawModule = chisel3.core.UserModule
+    type ExtModule = chisel3.core.ExtModule
```

Comment:
... this is in experimental ...
(GitHub diff apparently can't handle inner classes / objects / scopes when telling you the enclosing scope of a block of code)